src/pages/Missions/Missions.module.css

```diff
@@ -0,0 +1,69 @@
+.tableWrapper {
+  margin: 24px;
+}
+
+.table {
+  width: 100%;
+}
+
+.table,
+.tableHead,
+.tableData {
+  text-align: left;
+  padding: 8px;
+  border: 1px solid var(--lightWhite);
+  border-collapse: collapse;
+}
+
+.tableRow:nth-child(even) {
+  background-color: var(--whiteLighter);
+}
+
+.description {
+  font-size: 14px;
+  color: #4f4d4d;
+}
+
+.notMemberBadge {
+  display: inline-block;
+  min-width: 114px;
+  background: #7d7d7d;
+  color: var(--whiteLighter);
+  font-size: 12px;
+  padding: 4px 6px;
+  border-radius: 5px;
+  text-transform: uppercase;
+}
+
+.memberBadge {
+  display: inline-block;
+  min-width: 114px;
+  background: #0198e3;
+  color: var(--whiteLighter);
+  font-size: 12px;
+  padding: 4px 6px;
+  border-radius: 5px;
+  text-transform: capitalize;
+}
```

Comment:
- [ ] Good job so far but I think you should work a little bit on your button design, you should make sure your button sticks to the given design like the colors 😉 🙃 

Your design

<img width="974" alt="Screenshot 2023-04-14 at 01 02 15" src="https://user-images.githubusercontent.com/74501523/231896093-f46634c3-1048-4c18-96e5-a7486712711c.png">

Given design

<img width="1165" alt="space-travelers__missions" src="https://user-images.githubusercontent.com/74501523/231896127-09aaf77d-8e53-4813-baa6-1705b0b93bb2.png">